src/katgpucbf/xbgpu/engine.py

```diff
@@ -554,11 +547,11 @@ class XBEngine(DeviceServer):
     Data is passed between these items using :class:`asyncio.Queue`\s. The three
     processing functions are as follows:
 
-      1. :func:`_receiver_loop` - Receive chunks from network and initiate
-         transfer to GPU.
-      2. :func:`_gpu_proc_loop` - Perform the correlation operation.
-      3. :func:`_sender_loop` - Transfer correlated data to system RAM and then
-         send it out on the network.
+    - :func:`_receiver_loop` - Receive chunks from network and initiate
+        transfer to GPU.
```

Comment:
This is rendering funny, maybe due to the indentation.

Also `:func:` should be `:meth:` for methods.

Reply:
Please and thank you for the heads-up. Interestingly, I was able to correctly link gpu-proc and sender loops, but the receiver-loop just goes to bold. Is this the expected behaviour?
![image](https://github.com/ska-sa/katgpucbf/assets/25218138/324537a1-09b2-4933-8a90-bdae55cc3a0e)

Reply:
I think when I last looked, the `_receiver_loop` method wasn't appearing in the built documentation (and hence it can't be linked to), but I didn't investigate why.

Reply:
The mis-indentation is still messing up the rendering.